demo/chart.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable */
-var billboardDemo = {
+let billboardDemo = {
```

Comment:
Using ES6 syntax is good, but as having IE9 support this won't work for old browsers.
This is the reason of keeping old syntax.

Reply:
Roger that. Converting back to var notation.